site/source/tutorials/1040_dojo1_dijits/index.md

```diff
@@ -0,0 +1,191 @@
+
+{% instruction 'Create a `.dojorc` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/.dojorc' %}
+
+ To support Dojo 1 Dijits we have declared four external dependencies which will be copied during our build.  Additionally, some of those dependencies will be injected (loaded) into our bundle.  The first dependency provides configuration for the Dojo 1 loader (which we will create later).  The second dependency includes the `dojo` package and loads the Dojo 1 AMD loader which will then subsequently load the other Dojo 1 modules that are requested.  The third dependency is `dijit`, which are our Dojo 1 Dijits.  The final dependency is a modern flat theme for Dojo 1 Dijits.  We are injecting the _root_ CSS into our application bundle as well, which will ensure the CSS is loaded an available at run time.
+
+{% task 'Update the `tsconfig.json` configuration.' %}
+
+In order for us to integrate the TypeScript typings, we need to adjust two sections of our `tsconfig.json`.
+
+{% instruction 'Add `scripthost` to the `lib` section of `tsconfig.json`:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/tsconfig.json' lines:6-14 %}
+
+Some of the Dojo 1 typings reference typings from this library.
+
+{% instruction 'Add the Dijit module typings to the `include` section of `tsconfig.json`:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/tsconfig.json' lines:27-31 %}
+
+{% aside 'Information' %}
+If you are using your own custom Dijits, you should add your own TypeScript typings to the project.  The [`dojo/typings`](https://github.com/dojo/typings/) can provide you examples of how you might want to accomplish this.
+{% endaside %}
+
+By adding the modules to the scope of the TypeScript transpiler, we should have typing information available when attempting to import various Dijit modules.
+
+{% section %}
+
+## Configuring Dojo 1
+
+{% task 'Create loader configuration.' %}
+
+{% aside 'Information' %}
+The Dojo 1 loader configuration can be complex.  There is this Dojo 1 tutorial <em>[Configuring Dojo with dojoConfig](https://dojotoolkit.org/documentation/tutorials/1.10/dojo_config/index.html)</em> that explains how to configure Dojo 1.
+{% endaside %}
+
+In the previous step, we needed to include a configuration for our Dojo 1 loader.  For this tutorial, we only need to ensure that the Dojo 1 loader works asynchronously, because the Dojo 1 loader will default to synchronous operation, which would inhibit performance and cause issues with our Dojo 2 application.  The configuration could have any name.  Here we specify it as `/src/dojo1.cfg.js`.
+
+{% instruction 'Create a `/src/dojo1.cfg.js` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dojo1.cfg.js' %}
+
+{% task 'Ensure the application builds.' %}
+
+At this point, you should be able to build your application.
+
+{% instruction 'Run `dojo build` in the project root.' %}
+
+{% aside 'Reminder' %}
```

Comment:
Since asides are floated, it might look better to place this before the instruction tag on line 105 so that the aside is next to the `dojo build` command.